Objective-C/Tests/ReplicatorTest.m

```diff
@@ -1773,6 +1773,91 @@ - (void) testPullFilter {
 }
 
 
+- (void) testReplicationEventExpirationWithPush {
+    NSError* error;
+    CBLMutableDocument* doc = [[CBLMutableDocument alloc] initWithID: @"doc"];
+    [doc setString: @"Tiger" forKey: @"species"];
+    [doc setString: @"Hobbes" forKey: @"pattern"];
+    Assert([self.db saveDocument: doc error: &error]);
+    
+    XCTestExpectation* expectation = [self expectationWithDescription: @"Document expiry test"];
+    id token1 = [self.db addDocumentChangeListenerWithID: doc.id
```

Comment:
will change the variable name from token1 to meaningful name, `documentChangeToken`.